<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraCapture</class>
 <widget class="QMainWindow" name="cameraCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cameraCapture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LeftCamera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(116, 116, 116);
background-color: rgb(176, 176, 176);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RightCamera">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(116, 116, 116);
background-color: rgb(176, 176, 176);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SceneCamera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>650</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 2px;
border-style: solid;
border-color: rgb(116, 116, 116);
background-color: rgb(176, 176, 176);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenCamera">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenCamera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseCamera">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseCamera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Capture">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testgl">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>testgl</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
